packages/gcp/data_stream/loadbalancing_logs/sample_event.json

```
@@ -0,0 +1,136 @@
+{
+    "@timestamp": "2020-06-08T23:41:30.078Z",
+    "agent": {
+        "ephemeral_id": "1f7633a7-3410-4684-bb55-14b0bd0e2bd4",
+        "hostname": "docker-fleet-agent",
+        "id": "df142714-8028-4ef0-a80c-4eb03051c084",
+        "name": "docker-fleet-agent",
+        "type": "filebeat",
+        "version": "7.17.0"
+    },
+    "cloud": {
+        "project": {
+            "id": "PROJECT_ID"
+        },
+        "region": "global"
+    },
+    "data_stream": {
+        "dataset": "gcp.load_balancer",
```

As per https://github.com/elastic/integrations/pull/3493/files#r944481904, if changed there this should change to.
```suggestion
        "dataset": "gcp.loadbalancing_logs",
```